boot/zephyr/include/mcuboot_config/mcuboot_config.h

```diff
@@ -138,9 +138,38 @@
 
 #endif /* !__BOOTSIM__ */
 
+#if CONFIG_NRFX_WDT
+#include <nrfx_wdt.h>
+
+#define FEED_WDT_INST(id)                                    \
+    do {                                                     \
+        nrfx_wdt_t wdt_inst_##id = NRFX_WDT_INSTANCE(id);    \
+        for (uint8_t i = 0; i < NRF_WDT_CHANNEL_NUMBER; i++) \
+        {                                                    \
+            nrfx_wdt_channel_feed(&wdt_inst_##id,            \
```

Comment:
I missed one thing in the previous run. Since the nrfx_wdt driver is not initialized, `NRFX_ASSERT()` inside `nrfx_wdt_channel_feed()` will fail. This actually won't harm as long as assertions are not enabled. But you could instead call `nrf_wdt_reload_request_set(NRF_WDT#id, ...` and avoid that.